app/queries/bp_measures_query.rb

```diff
@@ -0,0 +1,22 @@
+class BPMeasuresQuery
+  def count(region, period_type, diagnosis: :hypertension, group_by: nil)
+    query = BloodPressure
+      .joins(:patient).merge(Patient.with_hypertension)
+      .group_by_period(period_type, :recorded_at, {format: Period.formatter(period_type)})
+      .where(facility: region.facilities)
```

Comment:
This code is ported from `ActivityService`? That _should_ ensure that these monthly numbers match the progress tab's monthly numbers. Should we use the repository in the progress tab right away to enforce consistency?

Reply:
I guess it gets hard because this service only deals with hypertensive patients. 🤔 Hmm, how should handle this divergence?

Reply:
> I guess it gets hard because this service only deals with hypertensive patients. 🤔 Hmm, how should handle this divergence?

Good question - I've been ignoring diabetes diagnosis for now, because just handling HTN is so much already.  I think we may want a new epic later to handle DM, whether its with a new repository or some objects that branch off of it.